packages/embark-library-manager/src/index.js

```diff
@@ -2,10 +2,15 @@ import { __ } from 'embark-i18n';
 import { dappPath, embarkPath } from 'embark-utils';
 var Npm = require('./npm.js');
 
+const DEPRECATIONS = {
+  'ipfs-api': 'ipfs-http-client'
```

Comment:
Nice work!  As part of the deprecation logic do we need to pair this with a default version of `ipfs-http-client` to use in case `ipfs-api` is found in `embark.json`?

Reply:
Great question!

This will use the default version for the replacement, so it'll forward the call to `version:get:ipfs-http-client`. That version comes from `this.versions`:

https://github.com/embark-framework/embark/pull/1647/files#diff-354526adef3a786756303636f591fd48R36

This would then be registered and used here:

https://github.com/embark-framework/embark/pull/1647/files#diff-354526adef3a786756303636f591fd48R80